README.md

```diff
@@ -1,8 +1,11 @@
 # Visual Studio Code Kubernetes Tools
 [![Build Status](https://travis-ci.org/Azure/vscode-kubernetes-tools.svg?branch=master)](https://travis-ci.org/Azure/vscode-kubernetes-tools)
 
-An extension for developers building applications to run in Kubernetes clusters,
-and for DevOps staff troubleshooting Kubernetes applications.  Features include:
+The extension for developers building applications to run in Kubernetes clusters.
+Works with any Kubernetes anywhere (Azure, Minikube, AWS, GCP and more!).
+Enables support for DevOps staff troubleshooting Kubernetes applications.
```

Comment:
I'm not keen on the way the "who and what is it for" is split up.  Can we restore the original first sentence. and move `Works with any Kubernetes anywhere...` to the second sentence?  (I think if anything that will make it _more_ prominent because it will be at the end of the paragraph rather than buried in the middle.)

Reply:
done.